src/components/SidebarComponents/Selection.vue

```diff
@@ -53,6 +65,10 @@ function addChart() {
 	storedChartNames.value.unshift(chartNameInput.value)
 	selected.value = chartNameInput.value
 	selectedChart.value = GenerateDefaultChart()
+	selectedChart.value.options.chartTitle = chartNameInput.value
+	emit('canRenderChart')
```

Comment:
canRenderChart is only needed at mounting stage and then shouldn't really need to change.